embulk-output-jdbc/src/test/java/org/embulk/output/jdbc/TestJdbcOutputPlugin.java

```diff
@@ -0,0 +1,39 @@
+package org.embulk.output.jdbc;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class TestJdbcOutputPlugin
+{
+    @Test
+    public void testCalculateSuffixLength()
+    {
+        {
+            try {
+                AbstractJdbcOutputPlugin.calculateSuffixLength(-1);
+            } catch (java.lang.AssertionError e) {
+                assert(true);
+            }
+        }
+        {
+            try {
+                AbstractJdbcOutputPlugin.calculateSuffixLength(0);
+            } catch (java.lang.AssertionError e) {
+                assert(true);
+            }
+        }
+        {
+            int i = AbstractJdbcOutputPlugin.calculateSuffixLength(1);
+            assertEquals(1, i);
+        }
```

Comment:
It was generating `"001"` for `1`, but it is going to generate `"1"` for `1`?

I thought:
* `001`, `002`, ..., `999`, `1000`, `1001`, ...

But is this actually :
* `1`, `2`, ..., `999`, `1000`, `1001`, ...

Reply:
@civitaspo @dmikurube 
If the minimum value for `suffixLength` is 3, we can maintain compatibility when `taskCount` is less than 1000.

Reply:
@dmikurube @hito4t Oh, I'm sorry. I had unintentionally crushed backward compatibility. I'll fix it!

Reply:
@dmikurube @hito4t Apply the review in https://github.com/civitaspo/embulk-output-jdbc/commit/c2ac8781b81b002b9be59f3ef405dee180c67bf6.